src/libcharon/plugins/vici/vici_cred.c

```diff
@@ -334,6 +334,7 @@ CALLBACK(load_token, vici_message_t*,
 	if (pin)
 	{	/* provide the pin in a temporary credential set to access the key */
 		shared = shared_key_create(SHARED_PIN, chunk_clone(chunk_from_str(pin)));
+		DBG4(DBG_CFG, "Loading shared PIN: %s", pin);
```

Comment:
The total reason for the proposed changes is to be able to validate that the loaded secret and ID is indeed what the user can see in the config file. So I'd like to log shared PINs, too. Also because that's what I'd expect if I configure log level 4 for CFG.

Reply:
Actually, I saw now that the PIN is cached in a separate, persistent credential set a bit later in the code. So maybe use this (more consistent with the other messages, also, they always start with lower case characters):

```suggestion
		DBG4(DBG_CFG, "loaded shared PIN for '%s': %s", hex, pin);
```